addons/website_form/static/src/js/website_form_editor.js

```diff
@@ -419,6 +419,9 @@ odoo.define('website_form_editor', function (require) {
             // Prevent saving of the status message  // TODO: would be better on Edit
             this.$target.find('#o_website_form_result').empty();
 
+            // Prevent saving disabled state of send button  // TODO: would be better on Edit
+            this.$target.find('.o_website_form_send').removeClass('disabled').removeAttr('disabled')
```

Comment:
@nle-odoo We can improve the fix in master but:

- Should not be done in cleanForSave but in the form widget "destroy" (which is called before entering edit mode) -> this is that widget which enables the form? Let it re-enable it too.
- Doing this in cleanForSave probably keeps the bug that the button is disabled during edit mode?
- What's the "TODO: would be better on Edit" about? :thinking:

Reply:
> What's the "TODO: would be better on Edit" about? 

just copying the language used above to be able to see that the 3 are related, in 14.0 this fix is moved to s_website_form.destroy with the other 2

> Doing this in cleanForSave probably keeps the bug that the button is disabled during edit mode?

I'm not sure how much this would affect.